tools/src/wyvern/stdlib/support/ArchAST.java

```diff
@@ -0,0 +1,6 @@
+package wyvern.stdlib.support;
+
+public class ArchAST {
+  public static final ArchAST utils = new ArchAST();
+  protected ArchAST() { }
+}
```

Comment:
I couldn't find any usages of this class. If this is not used, please remove it.